src/main/resources/reference.conf

```diff
@@ -981,3 +982,66 @@ search {
     }
   }
 }
+
+// OpenFDA FAERS Configuration
+
+openfda {
+    // NOTE - Each step seems to have a commong baseline path that should be refactored
+    step-root-input-path = ${common.input}"/fda"
+    step-root-output-path = ${common.output}"/fda"
+    // Source data
+    chembl-drugs { // --- DONE ---
+        format = "json"
+        path = ${drug.outputs.drug.path}"/*.json"
+    }
+    fda-data { // --- DONE ---
+        format = "json"
+        path = ${openfda.step-root-input-path}"/*.jsonl"
+    }
+    blacklisted-events { // --- DONE ---
+        format = "csv"
+        path = ${openfda.step-root-input-path}"/blacklisted_events.txt"
+        options = [
+            {k: "sep", v: "\\t"},
+            {k: "ignoreLeadingWhiteSpace", v: "true"},
+            {k: "ignoreTrailingWhiteSpace", v: "true"}
+        ]
+    }
+    meddra {
+        meddra-preferred-terms { // --- DONE ---
+            format = "csv"
+            // Change this to whatever location the data is sitting on
+            path = ${openfda.step-root-input-path}"/meddra/MedAscii/pt.asc"
+        }
+        meddra-low-level-terms { // --- DONE ---
+            format = "csv"
+            // Change this to whatever location the data is sitting on
+            path = ${openfda.step-root-input-path}"/meddra/MedAscii/llt.asc"
+        }
+    }
+    meddra-preferred-terms-cols = ["pt_code", "pt_name"]
+    meddra-low-level-terms-cols = ["llt_code", "llt_name"]
+    montecarlo { // --- DONE ---
+        permutations: 100
+        percentile: 0.95
+    }
+    sampling { // --- DONE ---
+        size = 0.1
+        enabled = false
+    }
+    outputs = {
+        fda-unfiltered {
+            format = "parquet"
```

Comment:
The output format should be set using the variable `common.output-format`.

Reply:
Done